lib/paths-cache.js

```diff
@@ -399,18 +400,31 @@ export default class PathsCache extends EventEmitter {
         .replace(/\./g, '\\.')
         .replace(/\*/g, '.*')
     );
-    const ignorePattern = '\'.*\\(' + ignorePatternsForFind.join('\\|') + '\\).*\'';
-
-    const cmd = 'find'
-    const args = [
-      '-L',
-      directoryPath + '/',
-      '-type', 'f',
-      '-not', '-regex', ignorePattern,
-    ];
-
-    const stdout = await spawn(cmd, args)
-    const files = _.compact(stdout.toString().split('\n'));
+
+    let files
+    if (process.platform === 'win32') {
+      const { stdout } = await execFile(
+        'dir',
+        [
+          '/A', // Show all files
+          '/S', // Include all subfolders
+          '/B', // Bare format (no heading, file sizes or summary)
+          directoryPath
+        ], {shell: true, maxBuffer: 10000*10000})
```

Comment:
Wouldn't it be better to use spawn here so we don't overflow the buffer?

Reply:
I plan to use native buffer instead of [`'bl'`](https://www.npmjs.com/package/bl) which is used by await-spawn. maxBuffer is just a limit that we can even set to inf. It does not actually allocate memory. It is like a protection/time out to stop receiving stuff. 

If we set the buffer to `buffer.constants.MAX_STRING_LENGTH` or `buffer.constants.MAX_LENGTH` the issue is removed.